lib/presentation/molecules/study_type_selection_molecule.dart

```diff
@@ -56,7 +56,7 @@ class _StudyTypeSelectionMoleculeState
     );
   }
 
-  Widget studyTypeRadioButton(void Function(TipType?)? onChanged,
+  Widget studyTypeRadioButton(void Function(TipType?) onChanged,
```

Comment:
Fixed unnecessary use of Nullability operator as requested in [this discussion](https://github.com/guyluz11/infinite_horizons/pull/24#discussion_r1597652883_
)

Reply:
To make it work change Radio onChanged to

          onChanged: (tipType) => onChanged(tipType ?? TipType.undefined),

Reply:
Yes, this is how I was going to implement it, I have not been getting the time to do it

Reply:
ok,

This is a small change and I want to release a new version, so I will do it.
Thanks